rdwatch/core/models/site_observation.py

```diff
@@ -96,25 +103,46 @@ def bulk_create_from_site_evaluation(
 
             constellation = constellation_map.get(feature.properties.sensor_name, None)
 
-            assert isinstance(feature.parsed_geometry, Polygon | MultiPolygon)
+            assert isinstance(feature.parsed_geometry, Polygon | MultiPolygon | Point)
 
-            geometry = (
-                feature.parsed_geometry
-                if isinstance(feature.parsed_geometry, MultiPolygon)
-                else MultiPolygon([feature.parsed_geometry])
-            )
-            for i, polygon in enumerate(geometry):
+            point = False
+            if isinstance(feature.parsed_geometry, Point):
+                point = True
+            if point is False:
+                geometry = (
+                    feature.parsed_geometry
+                    if isinstance(feature.parsed_geometry, MultiPolygon)
+                    else MultiPolygon([feature.parsed_geometry])
+                )
+                for i, polygon in enumerate(geometry):
+                    if feature.properties.current_phase:
+                        phase = feature.properties.current_phase[i]
+                    else:
+                        phase = 'Unknown'
+
+                    site_observations.append(
+                        cls(
+                            siteeval=site_eval,
+                            label=label_map[phase],
+                            score=feature.properties.score,
+                            geom=polygon,
+                            constellation=constellation,
+                            spectrum=None,
+                            timestamp=feature.properties.observation_date,
+                        )
+                    )
+            if point:
+                point = feature.parsed_geometry
                 if feature.properties.current_phase:
-                    phase = feature.properties.current_phase[i]
+                    phase = feature.properties.current_phase[0]
```

Comment:
Why this change?

Reply:
Sorry, it gets a bit confusing.

So there is a function that is used to interpret a string delimited list (by ',') of the current phase from the geoJSON.  I didn 't write this but I assume there is a reason for it:
https://github.com/ResonantGeoData/RD-WATCH/blob/rdwatch-point-support/rdwatch/core/schemas/site_model.py#L123-L130https://github.com/ResonantGeoData/RD-WATCH/blob/rdwatch-point-support/rdwatch/core/schemas/site_model.py#L123-L130

In the geom/polygon version, it would loop across the geometry (given that it could be a multi polygon)t and it looks like the code just took the phase in the string-separated list for the last value.  It's weird and I don't know the exact reason for this with multi polygons.  Maybe it was a way to support multiple phase activities without changing DB models in the past?

  For Points there shouldn't be multiple geometry so I just took the first value in the list.